checkstyle.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 <module name="Checker">
-  <module name="NewlineAtEndOfFile">
-    <property name="lineSeparator" value="lf"/>
-  </module>
```

Comment:
I'll keep this one: keeping a line at the end of a file is better for the diff, and at least we ensure that we won't have windows-style (or mac style ;) ) end of lines.

See: http://checkstyle.sourceforge.net/config_misc.html#NewlineAtEndOfFile